src/internal/connector/exchange/contacts_backup.go

```diff
@@ -0,0 +1,40 @@
+package exchange
+
+import (
+	"github.com/alcionai/corso/src/internal/connector/graph"
+	"github.com/alcionai/corso/src/pkg/services/m365/api"
+)
+
+var _ backupHandler = &contactBackupHandler{}
+
+type contactBackupHandler struct {
+	ac api.Contacts
+}
+
+func newContactBackupHandler(
+	ac api.Client,
+) contactBackupHandler {
+	acc := ac.Contacts()
+
+	return contactBackupHandler{
+		ac: acc,
+	}
+}
+
+func (h contactBackupHandler) itemEnumerator() addedAndRemovedItemGetter {
+	return h.ac
+}
+
+func (h contactBackupHandler) itemHandler() itemGetterSerializer {
+	return h.ac
+}
+
+func (h contactBackupHandler) NewContainerCache(
+	userID string,
+) (string, graph.ContainerResolver) {
+	return DefaultContactFolder, &contactFolderCache{
```

Comment:
why not populate the container cache in this call instead of returning the root folder and having the caller populate it?

Reply:
Minimizes logic changes in the PR.  I agree that it would be more sensible to populate it here.  Might look into this as a follow up or a future refactor.